rsmtool/rsmpredict.py

```diff
@@ -25,6 +28,151 @@
 from .writer import DataWriter
 
 
+def fast_predict(
+    input_features,
+    modeler,
+    df_feature_info,
+    df_postprocessing_params,
+    predict_expected=False,
+    logger=None,
+):
+    """
+    Compute predictions for a single instance against given model.
```

Comment:
Can this function be open to multiple instances?

Reply:
What would be the use case? The primary use for this function is in an engine scoring service which would only operate on a single instance at a time.

Reply:
We dont want any real-time prediction on batch instances? Sorry, I did not use rsmpredict much, so maybe my concepts are not that clear.

Reply:
Batch prediction is not really used in prediction. That's mostly only used in experimentation and model-building mode.